projects/fsastorefrontlib/src/occ/services/value-list/occ-value-list.service.spec.ts

```diff
@@ -44,55 +44,67 @@ describe('OccValueListService', () => {
   });
 
   describe('OccValueListService', () => {
-    it('should get values from value list API call', async(() => {
-      occValueListService.getValuesFromAPI(mockValueListPath).subscribe();
-      httpMock.expectOne((req: HttpRequest<any>) => {
-        return (
-          req.url === baseEndpoint + mockValueListPath && req.method === 'GET'
-        );
-      }, `GET method and url`);
-    }));
+    it(
+      'should get values from value list API call',
+      waitForAsync(() => {
```

Comment:
Why it this changed to waitForAsync? On develop it should be async() method

Reply:
Lint was throwing errors and warning that this method should be changed. Other files have waitForAsync as well.